src/types.rs

```diff
@@ -61,21 +57,8 @@ pub static GROUP_SIZE: usize = 8;
 pub static QUORUM_SIZE: usize = 6;
 
 pub type MessageId = u32;
-pub type NodeAddress = NameType; // (Address, NodeTag)
-pub type FromAddress = NameType; // (Address, NodeTag)
-pub type ToAddress = NameType; // (Address, NodeTag)
-pub type GroupAddress = NameType; // (Address, GroupTag)
-pub type SerialisedMessage = Vec<u8>;
-pub type IdNode = NameType;
-pub type IdNodes = Vec<IdNode>;
 pub type Bytes = Vec<u8>;
 
-#[derive(RustcEncodable, RustcDecodable)]
-struct SignedKey {
```

Comment:
This is a bit open for discussion.  @dirvine this was introduced at the beginning of rust-3, but as currently encryption is disabled, it might be okay for it to go.  I would vote for clearing it as we currently don't use it, but possible worth a look